molecule/docker/molecule.yml

```diff
@@ -0,0 +1,33 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: docker
+    image: replicajune/ubuntu-dind-supervisord
+    docker_host: "${DOCKER_HOST:-unix://var/run/docker.sock}"
+    privileged: true
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+      - /var/run/docker.sock:/var/run/docker.sock:rw
+      - /root/.ansible:/root/.ansible:rw
```

Comment:
Why is anything from `/root` needed to be mounted to run test?

Reply:
I've tried to avoid that a couple of times. I can't remeber if it was because of my setup (with vagrant) when molecule would be launch through a script from the Vagrantfile, being executed as root then.